sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerRequestTask.java

```diff
@@ -45,11 +45,14 @@ protected Void doInBackground(Void... params) {
             request.setVisibleInDownloadsUi(false);
         }
         long enqueueTime = System.currentTimeMillis();
-        long downloadId = downloadManager.enqueue(request);
-        if (isCancelled()) {
-            return null;
+        try {
+            long downloadId = downloadManager.enqueue(request);
+            if (!isCancelled()) {
+                mDownloader.onDownloadStarted(downloadId, enqueueTime);
+            }
+        } catch (IllegalArgumentException e) {
+            AppCenterLog.error(LOG_TAG, "Failed to download: Download Manager is disabled.");
```

Comment:
I think we need to propagate the error to mDownloader.onDownloadError(), so `onError` callback of the **ReleaseDownloader.Listener** can be called.

Reply:
That's a good point